app/i18n/es.json

```diff
@@ -603,7 +604,8 @@
     "payFromUSD": "Pago desde $USD",
     "success": "El pago se ha enviado correctamente.",
     "title": "Enviar Bitcoin",
-    "totalExceed": "total excede su saldo"
+    "totalExceed": "total excede su saldo",
```

Comment:
Not part of your change but I just happened to notice that this is only used in the `SendBitcoinConfirmationScreen` so I think this is an unnecessary key - could be removed.